sdk/keyvault/azure-security-keyvault-keys/src/main/java/com/azure/security/keyvault/keys/KeyClientBuilder.java

```diff
@@ -63,7 +64,7 @@
     private final List<HttpPipelinePolicy> policies;
     private TokenCredential credential;
     private HttpPipeline pipeline;
```

Comment:
Need to update `endpoint` to `vaultUrl` in above docs.

Reply:
updated